api/v1/configurationpolicy_types.go

```diff
@@ -157,8 +157,14 @@ type ConfigurationPolicySpec struct {
 	// 'matchLabels' and/or 'matchExpressions' are, 'include' will behave as if ['*'] were given. If
 	// 'matchExpressions' and 'matchLabels' are both not provided, 'include' must be provided to
 	// retrieve namespaces.
-	NamespaceSelector  Target             `json:"namespaceSelector,omitempty"`
+	NamespaceSelector Target `json:"namespaceSelector,omitempty"`
+	// 'object-templates' and 'object-templates-raw' are arrays of objects for the configuration
+	// policy to check, create, modify, or delete on the cluster. 'object-templates' is an array
+	// of objects, while 'object-templates-raw' is a string containing an array of objects in
+	// YAML format. Only one of the two object-templates variables can be set in a given
+	// configurationPolicy.
 	ObjectTemplates    []*ObjectTemplate  `json:"object-templates,omitempty"`
+	ObjectTemplatesRaw string             `json:"object-templates-raw,omitempty"`
```

Comment:
Could you add a description so that `kubectl explain ConfigurationPolicy.spec.object-templates-raw` will return something helpful? Even if it's just a copy of the above comment or just say to look at the docs for `object-templates`.